README.md

```diff
@@ -9,4 +9,12 @@ Utilities used for experiments and testing with Quarkus
 
 - `create_colorful_diff.sh` utility to create  the colorful diff between two files.
 - Java program that prints dependencies of added artifacts \
-`mvn clean install exec:java -Dquarkus.maven.dir="path_to_dir" -Dquarkus.new-artifacts-list="added_artifacts.txt"`
\ No newline at end of file
+`mvn clean install exec:java -Dquarkus.maven.dir="path_to_dir" -Dquarkus.new-artifacts-list="added_artifacts.txt"`
+
+
+### quarkus-extensions-with-dependency
+
+Creates PDF with Quarkus extensions contain given dependency (quarkus-vertx, quarkus-resteasy etc.) 
+
+- Usage:`get-extensions.sh DEPENDENCY_NAME [OUTPUT_FILE_NAME.pdf]` 
+- Result will be written by default to the PDF document `Quarkus-extensions-with-*.pdf` or to the file with your custom name
```

Comment:
Also, it would be nice to include in the README the sh command that you describe in the PR description...`./get-extensions.sh io.quarkus:quarkus-vertx [name-file].pdf` or whatever.

Reply:
Extend it about example command with vertx